addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2110,6 +2110,16 @@ export class OdooEditor extends EventTarget {
     toolbarShow() {
         this._updateToolbar(true);
     }
+
+    closeDropDowns() {
+        this.toolbar.querySelectorAll("ul.dropdown-menu.show").forEach((e) => {
+            const dropdownButton = e.parentElement.querySelector("button[data-toggle='dropdown'], div[data-toggle='dropdown']");
+            if (dropdownButton) {
+                dropdownButton.click();
```

Comment:
Was there no way to accomplish this programmatically instead of triggering a click?